src/helperFunctions/formatProjectDataHelper.js

```diff
@@ -1,17 +1,36 @@
 export const formatProjectDataHelper = (project) => {
-  const projectName = project.records[0].project_name
-  const years = [
-    ...new Set(project.records.map((record) => record.sample_date.substring(0, 4))),
-  ].sort((a, b) => a.localeCompare(b))
+  const { records } = project
+  const data = {
+    years: new Set(),
+    countries: new Set(),
+    organizations: new Set(),
+    surveyCount: records.length,
+    transects: 0,
+  }
+
+  records.forEach((record) => {
+    data.years.add(record.sample_date.substring(0, 4))
+    data.countries.add(record.country_name)
+    if (record.tags) {
+      record.tags.forEach((tag) => data.organizations.add(tag.name))
+    }
+    const sumOfSampleUnitCounts = Object.values(record.protocols).reduce((sum, protocol) => {
+      return sum + protocol.sample_unit_count
+    }, 0)
+    data.transects += sumOfSampleUnitCounts
+  })
+
+  const years = [...data.years].sort((a, b) => a.localeCompare(b))
   const formattedYears = years.length === 1 ? years[0] : `${years[0]}-${years[years.length - 1]}`
-  const countries = [...new Set(project.records.map((record) => record.country_name))]
-    .sort((a, b) => a.localeCompare(b))
-    .join(', ')
-  const organizations = [
-    ...new Set(project.records.map((record) => record.tags?.map((tag) => tag.name)).flat()),
-  ]
-    .sort((a, b) => a.localeCompare(b))
-    .join(', ')
-  const siteCount = [...new Set(project.records.map((record) => record.site_name))].length
-  return { projectName, formattedYears, countries, organizations, siteCount }
+  const countries = [...data.countries].sort((a, b) => a.localeCompare(b)).join(', ')
+  const organizations = [...data.organizations].sort((a, b) => a.localeCompare(b)).join(', ')
```

Comment:
Could it just be the following?
  `const countries = data.countries.sort((a, b) => a.localeCompare(b)).join(', ')`

Reply:
Countries and organizations are a set (because of the uniqueness feature of sets) and sets can't be sorted. It's kind of like how objects don't have a sort method and you can't really sort the keys. So the spread operator converts the set into an array, and then I can use the .sort() array method on it